src/components/donation/Steps/Submit/DAFCheckout/ManualDonation.tsx

```diff
@@ -58,7 +58,7 @@ export default function ManualDonation(props: DafCheckoutStep) {
           profileUrl,
           props.details.amount
         )}
-        className="mt-1.5 btn btn-blue rounded-full px-4 py-2 justify-self-end text-xs font-normal"
+        className="mt-1.5 btn btn-blue bg-[--accent-primary] enabled:hover:bg-[--accent-primary] rounded-full px-4 py-2 justify-self-end text-xs font-normal"
```

Comment:
hover color is set to be the same as base color as its another problem to determine lighter (simply decreasing opacity is suboptimal) color base on a given accent